redi-connect-front/src/components/templates/LoggedIn.scss

```diff
@@ -0,0 +1,17 @@
+@import "~bulma/sass/utilities/_all";
+@import "../../styles/_variables.scss";
+
+@include mobile() {
+  .logged-in {
```

Comment:
This belongs only to side-menu so add it to the side menu please

Reply:
done